javascript/javascript2/week2/hyfBay/main.js

```diff
@@ -0,0 +1,114 @@
+"use strict";
+console.log("Script loaded");
+
+const products = getAvailableProducts();
+const productsUl = document.querySelector("ul");
+const sortType = document.getElementById("sort-selection");
+//dispaly products
+function renderProducts(products) {
+  // clear list before displaying new list
+  clearList();
+
+  //create list of countries
+  products.forEach((product) => {
+    let shipsTo = "";
+    product.shipsTo.forEach((country) => (shipsTo += `<li>${country}</li>`));
+
+    const li = document.createElement("li");
+    li.className = "prduct-display";
+    li.innerHTML = `<ul>
+    <li>${product.name}</li>
+    <li> ${product.price}</li> 
+    <li> ${product.rating}</li>
+    <li> <ul>${shipsTo}</ul></li>
+    </ul>`;
+    productsUl.appendChild(li);
+  });
+}
+// display whole list first time
+renderProducts(getSorted(products));
+
+//display products matches to names intered by user
+const searchInputField = document.getElementById("serach-input");
+
+searchInputField.addEventListener("keyup", () =>
+  renderProducts(getSorted(getSearchProductByName(products)))
```

Comment:
```suggestion
const searcherProducts = getSearchProductByName(products);
const sortedProducts = getSorted();
 renderProducts(sortedProducts)
```

Make your code easy to read for other developers :)